plugins/inputs/intel_powerstat/README.md

```diff
@@ -11,18 +11,24 @@ to take preventive/corrective actions based on platform busyness, CPU temperatur
 ```toml
 # Intel PowerStat plugin enables monitoring of platform metrics (power, TDP) and per-CPU metrics like temperature, power and utilization.
 [[inputs.intel_powerstat]]
-  ## All global metrics are always collected by Intel PowerStat plugin.
+  ## Some package metrics are collected by Intel PowerStat plugin by default.
+  ## User can choose which per-Core metrics are monitored by the plugin in package_metrics array.
+  ## No array means default metrics per-Core will be collected by the plugin.
+  ## Empty array means no per-Core metrics will be collected by the plugin.
```

Comment:
I haven't gotten to this code yet, but how are you determining if the option is not an array versus empty?

Reply:
If we do not have the "package_metrics" comparison in the configuration, we use the default settings. If such an option appears in the configuration, first we set all values ​​to false, and then we set the values ​​specified by the user in the configuration to true
function parsePackageMetricsConfig() in file intel_powerst.go

Reply:
ok I see what you are saying; let's re-word this to be more clear:

```
  ## The user can choose which per-core metrics are monitored by the plugin with the package_metrics setting:
  ## * The default, will collect <TODO: put the default collected metrics here>
  ## * Setting this value to an empty array means no per-core metrics will be collected
  ## * Finally, a user can specify individual metrics to capture from the following list <TODO: put the list here>
```

Reply:
Readme has been corrected